getting_started/brainvision.md

```diff
@@ -35,7 +35,7 @@ For example, see this excerpt from a BrainVision header file (.vhdr):
     ; Data synthesized by MNE-BIDS
 
     [Common Infos]
-    DataFile=test.eeg
+    DataFile=test.dat
```

Comment:
I think that this should stay .eeg -> generically the raw files are stored as .eeg, only upon export they get the extension '.dat', as far as I know. Or is it different on your end?

Reply:
No, you are right, this should stay at .eeg then :)